pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -112,6 +115,22 @@ var (
 
 	caRSAKeySize = env.RegisterIntVar("CITADEL_SELF_SIGNED_CA_RSA_KEY_SIZE", 2048,
 		"Specify the RSA key size to use for self-signed Istio CA certificates.")
+
+	// CaAddrENV When this env is defined, it's require 'custom-ca.crt','custom-ca.key' and 'root-cert.pem'
+	// defined within 'cacerts' k8s secret
+	// 'custom-ca.crt' and 'custom-ca.key' are used to secure connection between IstioD and Custom CA
+	// 'custom-ca.crt' and 'custom-ca.key' are configurable by CUSTOM_CERT_NAME, CUSTOM_KEY_NAME env
+	CaAddrENV = env.RegisterStringVar("CUSTOM_CA_ADDR", "",
+		"Custom CA address to sign Workload Certificate")
+	// CustomCACertName file name of Custom CA TLS client certificate
+	CustomCACertName = env.RegisterStringVar("CUSTOM_CA_CERT_NAME",
```

Comment:
too much - maybe start with a directory where we use files with the normal pattern  (i.e. same we use for 
ingress certs, etc )?